experimental/emulated_vault/README

```diff
@@ -0,0 +1,23 @@
+The emulated_vault module supplies a HTTP server mimicking RIAK behavior for the usage as traffic-control vault.
+The server may use different type of persistent storage (e.g. file-system), using the proper adaptor.
+Basic requirements: Centos ver >= 7; Python
+
+In order to deploy the code on a server please:
+1. Copy the module files to the server's root
+2. Add the certificate and key to you favorite path
+3. Adjust opt/emulated_vault/conf/cfg.json - pointing at your certificate and key
+4. chkconfig the service
+
+Logs may be found under /opt/emulated_vault/var/log
+
+Developers notes:
+------------------------
```

Comment:
Is this a Markdown file? if so, this isn't how you do a heading in markdown, those are done like:
```markdown
# Developer's Notes
```
if this is meant to be a reStructuredText file, then the length of the underline must match the length of the title itself - and the level of hyphens is improper, should probably be <kbd>=</kbd>.

Either way, you wanna have have a top-level title for this README.

Reply:
Ack
Thanks

Reply:
You can do a header in Markdown like this. It's uncommon, but it's valid, in both the original spec (https://daringfireball.net/projects/markdown/syntax#header) and GHM (https://github.github.com/gfm/#setext-heading).